app/services/analytics_events.rb

```diff
@@ -4125,12 +4125,17 @@ def multi_factor_auth_added_phone(
   # Tracks when the user has added the MFA method piv_cac to their account
   # @param [Integer] enabled_mfa_methods_count number of registered mfa methods for the user
   # @param [Boolean] in_account_creation_flow whether user is going through creation flow
-  def multi_factor_auth_added_piv_cac(enabled_mfa_methods_count:, in_account_creation_flow:,
-                                      **extra)
+  # @param ['piv_cac'] method_name Authentication method added
+  def multi_factor_auth_added_piv_cac(
+    enabled_mfa_methods_count:,
+    in_account_creation_flow:,
+    method_name: :piv_cac,
```

Comment:
With the event being `mult_factor_auth_added_piv_cac`, what information would be captured in `method_name`?

Reply:
Yeah, I'm honestly not really sure I see the value in logging `method_name`. I kept it here just for parity with how it worked previously. I could imagine a potential query where someone filters like `filter properties.event_properties.method_name = 'piv_cac'` without an event name, since some other events follow this pattern as well (e.g. `'Multi-Factor Authentication: Added phone'`). But I kinda doubt that's common? We could create a follow-up ticket to investigate if we're querying that way and remove the redundant event property if unused.

Reply:
And to clarify, this change was necessary because it was flagged by [our undocumented analytics parameters checker](https://github.com/18F/identity-idp/blob/9ab3079a2ba17550dddb9b068e6ff64bc9ae53f6/spec/support/fake_analytics.rb#L73-L146). It wasn't flagged previously because [`sign_in_spec.rb` still exempts itself from these checks](https://github.com/18F/identity-idp/blob/9ab3079a2ba17550dddb9b068e6ff64bc9ae53f6/spec/features/users/sign_in_spec.rb#L3), but creating a new feature test file without the exemptions caused it to be caught.

Reply:
This is just the default, was added a long time ago but I do wonder if we still need it.